showcase/src/app/welcome/components/news-item/_news-item.component.scss

```diff
@@ -1,5 +0,0 @@
-.section-border {
```

Comment:
Sorry, I didn't mean to replace this file with a xxx-theme.scss one but to move all the "themable" properties to the xxx-theme.scss file and leave in this one the rest